### DISCUSSION TITLE
Failed download of common actions on "Set up" job

### DISCUSSION THREAD

**Discussion by marino-t** [community/community#25774] [Actions]
<p>I don’t understand why these errors occur.<br>
12 minutes for set up :disappointed_relieved:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/0/7083f7281fece79d853821c653e2b57e7ff09b0d.png" data-download-href="/uploads/short-url/g3md6wMdh4rp2R2SS7unZWzoDet.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/96420191/181093197-0aba03ba-2f82-479a-b4f4-9290b383b0d9.png" alt="image" data-base62-sha1="g3md6wMdh4rp2R2SS7unZWzoDet" width="689" height="434" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/0/7083f7281fece79d853821c653e2b57e7ff09b0d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1292×814 63.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by aman-deep739** [comment]
<p>facing the same issue since last 3 hours… eventually deployment times out</p>

**Comment by solidi** [comment]
<p>Hi, I am facing the same problem since yesterday. The jobs do eventually execute, but action build time is increased by one thousand percent. Includes setup job delays, failure to download packages like <code>actions/checkout@v2</code>, and hangups on git clones/git checkouts.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/1/11868e7a235f91b0566b5e1c2a485e056d843c2c.png" data-download-href="/uploads/short-url/2v2oZr0p3lKeuahobqdp4iwTZfe.png?dl=1" title="download-problems" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/729372/181093200-885e47d9-baa0-4a33-835a-701182f502c7.png" alt="download-problems" data-base62-sha1="2v2oZr0p3lKeuahobqdp4iwTZfe" width="649" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/11868e7a235f91b0566b5e1c2a485e056d843c2c_2_649x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/1/11868e7a235f91b0566b5e1c2a485e056d843c2c_2_973x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/1/11868e7a235f91b0566b5e1c2a485e056d843c2c_2_1298x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/11868e7a235f91b0566b5e1c2a485e056d843c2c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">download-problems</span><span class="informations">2528×1946 425 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>See an example run: <a href="https://github.com/solidi/hl-mods/actions/runs/1626103336" class="inline-onebox" rel="noopener nofollow ugc">Strip submodules · solidi/hl-mods@ad0b388 · GitHub</a></p>

**Comment by ghost** [comment]
<p>Same issue for us. Github Action Runs randomly getting cancelled.</p>

**Comment by hoblin** [comment]
<p>Same problem. Runners do not pick up jobs or fail without any logs. GitHub status is still green though</p>
<pre><code class="lang-auto">Can't find any online and idle self-hosted or hosted runner in the current repository, account/organization that matches the required labels: 'ubuntu-latest'
Waiting for a self-hosted or a hosted runner to pickup this job...
</code></pre>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/d/7d5347a5a7b8f4f1ae01e78a7223b6f71c3fe3d4.png" data-download-href="/uploads/short-url/hSG6PfM3uQFSlsMoFFwpKPb8syo.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/28090/181093201-7f2d27a9-2b79-4efc-a078-9db6859e9769.png" alt="image" data-base62-sha1="hSG6PfM3uQFSlsMoFFwpKPb8syo" width="690" height="281" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d5347a5a7b8f4f1ae01e78a7223b6f71c3fe3d4_2_690x281.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d5347a5a7b8f4f1ae01e78a7223b6f71c3fe3d4_2_1035x421.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d5347a5a7b8f4f1ae01e78a7223b6f71c3fe3d4_2_1380x562.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/d/7d5347a5a7b8f4f1ae01e78a7223b6f71c3fe3d4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1846×754 45.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by pboymt** [comment]
<p>Same problem and did not occur again until action was retried many times.</p>

**Comment by pfeerick** [comment]
<p>Looks to be the same as <a href="https://github.community/t/read-timeouts-and-errors-from-github-urls/219615" class="inline-onebox">Read timeouts and errors from GitHub URLs</a></p>

**Comment by jfullstackdev** [comment]
<p>might still be related to that problem, it’s still on-going,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270.png" data-download-href="/uploads/short-url/nXlBh3eVA0o0LvtBfcN4YAL2yTS.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181093205-aa2a1462-dbc0-47ce-987b-a6a299e61d28.png" alt="image" data-base62-sha1="nXlBh3eVA0o0LvtBfcN4YAL2yTS" width="690" height="128" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270_2_690x128.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270_2_1035x192.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270_2_1380x256.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/7/a7e771532dd328aef9bce3074cb314aabcb29270_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1540×287 42.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p><a href="https://www.githubstatus.com/" rel="noopener nofollow ugc">https://www.githubstatus.com/</a></p>

**Comment by aman-deep739** [comment, marked as answer]
<p>looks like its back up</p>
